### PR TITLE
Add argocd project for peribolos service

### PR DESCRIPTION
**Pull request by Shreyanand**
For issue https://github.com/open-services-group/peribolos-as-a-service/issues/32

@harshad16 @tumido

### CODE REVIEW

File: argocd/overlays/moc-infra/projects/peribolos-service.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: peribolos-as-a-service
+  labels:
+    project-template: global
+spec:
+  destinations:
+    - namespace: '*'
+      name: 'smaug'
```
Comment (tumido) [inline argocd/overlays/moc-infra/projects/peribolos-service.yaml:10]: This grants the project access to any namespace on smaug, we don't want that.

File: argocd/overlays/moc-infra/configs/argo_rbac_cm/policy.csv [hunk 2]
```diff
@@ -30,3 +30,5 @@ g, sre, role:standard-user
 g, fybrik, role:standard-user
 g, tremor-demo, role:standard-user
 g, osc-admins, role:standard-user
+g, peribolos-as-a-service, role:standard-user
```
Comment (tumido) [inline argocd/overlays/moc-infra/configs/argo_rbac_cm/policy.csv:33]: I think there was some reason to *not* include the EOL at EOF here. cc @HumairAK for confirmation.

File: argocd/overlays/moc-infra/projects/peribolos-service.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: peribolos-as-a-service
```
Comment (tumido) [inline argocd/overlays/moc-infra/projects/peribolos-service.yaml:4]: Filename should match this name here, I think. `peribolos-service.yaml` file vs `peribolos-as-a-service` project.

**Review by tumido — CHANGES_REQUESTED** [review]

File: argocd/overlays/moc-infra/configs/argo_rbac_cm/policy.csv (hunk 2 above)
Comment (HumairAK) [reply argocd/overlays/moc-infra/configs/argo_rbac_cm/policy.csv:33]: yeah there was...but the reason is escaping me, so just be safe and keep it without eol

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**